infra/terraform/modules/data_access/common/sentry/sentry/sentry.py

```diff
@@ -0,0 +1,26 @@
+'''
+Environment variables:
+ - ENV, e.g. "dev", "alpha", etc...
+ - SENTRY_DSN, Sentry DSN
+'''
+
+import os
+
+from raven import Client as Sentry
+from raven.transport.http import HTTPTransport as SentryHTTPTransport
+
+
+def catch_exceptions(fn):
+    def wrapped(*args, **kwargs):
+        try:
+            fn(*args, **kwargs)
+        except Exception:
+            client = Sentry(
+                os.environ["SENTRY_DSN"],
+                transport=SentryHTTPTransport,
+                environment=os.environ["ENV"],
+            )
+            client.captureException()
+            raise
+
+    return wrapped
```

Comment:
I've seen this before and I get why it makes sense in the beginning. But then this soon become a dumping ground for "anything common". I like to keep packages small with the potential for having them potentially reused independently one day.